gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/keyedgrouping/GroupingKeys.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.keyedgrouping;
+
+import org.apache.flink.api.java.tuple.Tuple;
+import org.gradoop.common.model.api.entities.Attributed;
+import org.gradoop.common.model.api.entities.Element;
+import org.gradoop.common.model.api.entities.Labeled;
+import org.gradoop.flink.model.api.functions.KeyFunction;
+import org.gradoop.flink.model.api.functions.KeyFunctionWithDefaultValue;
+import org.gradoop.flink.model.impl.operators.keyedgrouping.keys.CompositeKeyFunction;
+import org.gradoop.flink.model.impl.operators.keyedgrouping.keys.LabelKeyFunction;
+import org.gradoop.flink.model.impl.operators.keyedgrouping.keys.LabelSpecificKeyFunction;
+import org.gradoop.flink.model.impl.operators.keyedgrouping.keys.PropertyKeyFunction;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A factory class for creating instances of commonly used grouping key functions.
+ */
+public class GroupingKeys {
```

Comment:
I used plural here since this class is a factory for a variety for key functions.  Those kinds of classes use plural in the java standard library too, like `java.util.Collections`, `java.util.Objects` or `java.util.stream.Collectors`

Reply:
'K